backend/framework/qlf/dashboard/bokeh/qaxwsigma/main.py

```diff
@@ -36,23 +37,60 @@ def __init__(self, process_id, arm, spectrograph):
             self.selected_spectrograph = spectrograph
 
     def load_qa(self):
+
         cam = self.selected_arm+str(self.selected_spectrograph)
+        # try:
```

Comment:
If you are not going to use it just delete it. We can recover code from git history